lib/fbe/octo.rb

```diff
@@ -800,7 +957,13 @@ def workflow_run_job(_repo, job)
         completed_at: '2024-08-18T08:20:17Z'
       }
     ].select { |json| json[:id] == job }
-    result.first
+    result.first || {
```

Comment:
@Suban05 It will probably be better if you rewrite it as:
```ruby
[
  {
    id: 28_907_016_501,
    run_id: 10_438_531_072,
    name: 'make',
    started_at: '2024-08-18T08:04:44Z',
    completed_at: '2024-08-18T08:20:17Z'
  },

 # ...
].find { |json| json[:id] == job } || {
  id: job,
  run_id: 1234,
  name: 'run job',
  started_at: '2024-08-18T08:04:44Z',
  completed_at: '2024-08-18T08:20:17Z'
}
```